### PR TITLE
Automatic Table Creation and Insert Sample Data

### PR DESCRIPTION
**Pull request by wakatakeru**
## Summary

Implement automatic table creation and sample data insertion. 
In this change, an SQL script is placed in `docker-entrypoint-initdb.d` in the container to create the table and insert the sample data.

## Changes

- Update: `docker-compose.yml`
- Create: `sql/01.create-user-table.sql`
- Create: `sql/02.insert-initial-users.sql`

### CODE REVIEW

File: sql/01.create-user-table.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+CREATE TABLE users (id VARCHAR(64), name VARCHAR(128), password_hash TEXT, email TEXT);
```
Comment (yamakenji24) [inline sql/01.create-user-table.sql]: Thought the maximum length of the email declared in RFC was 256.

**Review by yamakenji24 — APPROVED** [review]
Just some comments. 
LGTM

File: sql/01.create-user-table.sql (hunk 1 above)
Comment (wakatakeru) [reply sql/01.create-user-table.sql]: Indeed. It seems that it must be 254 characters or less.
I'm going to fix it.

https://tools.ietf.org/html/rfc5321#section-4.5.3

**Review by yamakenji24 — APPROVED** [review]
LGTM

**Pull request merged**